gtdynamics/utils/Initializer.cpp

```diff
@@ -168,8 +170,10 @@ Values Initializer::InitializeSolutionInterpolationMultiPhase(
         robot, link_name, pose, wTl_t[i], curr_t, ts[i], dt, gaussian_noise,
         contact_points);
 
-    for (auto&& key_value_pair : phase_vals) {
-      init_vals.tryInsert(key_value_pair.key, key_value_pair.value);
+    for (auto&& key : phase_vals.keys()) {
```

Comment:
init_vals.insert_or_assign(phase_vals);

Reply:
Same deal. `insert_or_assign` doesn't do the same thing as `tryInsert`. 😢